artifacts/centos/centos.go

```diff
@@ -100,6 +104,34 @@ func (c *centos) Inspect() (*api.ArtifactDetails, error) {
 
 }
 
+func (c *centos) VMI(imgRef string) *kvirtv1.VirtualMachineInstance {
+	options := []libvmi.Option{
```

Comment:
We really want to test what we also post on e.g. here as example: https://quay.io/repository/containerdisks/fedora.

The only difference should the the ssh-key and the temporary image reference.